README.md

```diff
@@ -5,6 +5,7 @@
 [![Build Status](https://github.com/rusqlite/rusqlite/workflows/CI/badge.svg)](https://github.com/rusqlite/rusqlite/actions)
 [![dependency status](https://deps.rs/repo/github/rusqlite/rusqlite/status.svg)](https://deps.rs/repo/github/rusqlite/rusqlite)
 [![Latest Version](https://img.shields.io/crates/v/rusqlite.svg)](https://crates.io/crates/rusqlite)
+[![Gitter](https://badges.gitter.im/diesel-rs/diesel.svg)](https://gitter.im/rusqlite/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
```

Comment:
Presumably this shouldn't reference diesel-rs/diesel?

Reply:
Yes, it shouldn't.

Reply:
Err, it does, to `https://badges.gitter.im/diesel-rs/diesel.svg`

Reply:
@thomcc Yes, it shouldn't.

Reply:
@thomcc I meant that the code isn't right.

Reply:
I've taken diesel svg because the default one(https://blog.gitter.im/2014/01/27/badgers/) looked bad. Basically it will just give a gitter chat button with link to rusqlite chat.
Here the example of this button in diesel/diesel.rs: https://github.com/diesel-rs/diesel

Reply:
Using `https://badges.gitter.im/rusqlite.svg` looks the same to me...

Reply:
@thomcc Okay, I changed the files.